resources/views/service/edit_registration.blade.php

```diff
@@ -93,22 +93,43 @@
                 </div>
                 <button class="long-button" type="submit">Save Changes</button>
             </div>
-
             <div class="col collect">
                 <div>
                     <img src="{{ asset('assets/info-light.svg') }}" name="logo">
                     <h2>This family</h2>
                 </div>
                 <div>
-                    <p>This family may collect <strong>{{ $family->entitlement }}</strong> per week:</p>
+                    <p>This family:</p>
                     <ul>
-                        @foreach( $family->getCreditReasons() as $credits )
-                            <li>
-                                <strong>{{ $credits['reason_vouchers'] }} {{ str_plural('voucher', $credits['reason_vouchers']) }}</strong>
-                                as {{ $credits['count'] }} {{ str_plural($credits['entity'], $credits['count']) }}
-                                currently "{{ $credits['reason'] }}"
-                            </li>
-                        @endforeach
+                        <li>
+                            Should collect 
+                            <strong>
+                                {{ $family->entitlement }}
+                            </strong> 
+                            per week
+                        </li>
+                        <li>
+                            Has 
+                            <strong>
+                                {{ count($family->children) }}
+                            </strong> 
+                            children on the scheme 
```

Comment:
"children registered"?

I think that, technically, kids above 5 years are not "on scheme", becuase they're too old.

Reply:
true! Will amend and push up now